volatility/framework/plugins/mac/psaux.py

```diff
@@ -86,11 +86,12 @@ def _generator(self, tasks: Iterator[Any]) -> Generator[Tuple[int, Tuple[int, st
             yield (0, (task.p_pid, task_name, task.p_argc, args_str))
 
     def run(self) -> renderers.TreeGrid:
-        filter_func = tasks.Tasks.create_pid_filter([self.config.get('pid', None)])
+        filter_func = pslist.PsList.create_pid_filter([self.config.get('pid', None)])
+        list_tasks = pslist.PsList.get_list_tasks(self.config.get('pslist_method', pslist.PsList.pslist_methods[0]))
```

Comment:
can pslist.PsList.pslist_methods[0] just be 'tasks' live above?

Also, why does this one do config.get('pslist_method', ...), but then the _generator does  config.get('method',...) ?